src/Shared/ProblemDetails/ProblemDetailsDefaults.cs

```diff
@@ -63,11 +76,35 @@ internal static class ProblemDetailsDefaults
             "Unprocessable Entity"
```

Comment:
Also, this is now called `Unprocessable Content`. I'm not sure what the best way to go forward with that is, though 😅

Reply:
I've been trying to reach someone on the team in order to avoid shipping *new* APIs in .NET 7 with the obsolete name, but haven't been successful 😔

Reply:
See https://github.com/dotnet/aspnetcore/issues/42302 🦗🦗🦗

Reply:
Yeah I spotted it had a new name when looking through the docs to get the new link, but that seemed a bigger change that just updating some URLs so I left it alone. Again, I could easily update it if the team want it updated too.

Reply:
Yeah, this PR should probably be focused on more titles and update links :)

Reply:
Yeah, let just update the link and title here.